pages/common/viu.md

```diff
@@ -0,0 +1,24 @@
+# viu
+
+> A small command-line application to view images from the terminal.
+> More information: <https://github.com/atanunq/viu>.
+
+- Render an image or GIF:
+
+`viu {{path/to/file}}`
+
+- Render an image or GIF from the internet using `curl`:
+
+`curl -s {{https://example.com/image.png}} | viu -`
+
+- Render a transparent image with a transparent background:
+
+`viu -t {{path/to/file}}`
+
+- Render an image with a specific width and height in pixels:
+
+`viu -w {{width}} -h {{height}} {{path/to/file}}`
+
+- Render PNG or GIF files with their original names:
+
+`viu -n {{*.{png,gif}}}`
```

Comment:
The `*.png` is a feature of the shell, not of `viu`, so we should focus on the `-n` flag here.
```suggestion
- Render an image or GIF and display its file name:

`viu -n {{path/to/file}}`
```